code_review.txt

```diff
@@ -0,0 +1 @@
+<mei write here>
```

Comment:
get() and count() are two modules in both db_storage.py and file_storage.py. Those get() returns an object based on the class and its ID. If the class is not exist, it returns None. Those count() returns count of number of instances of an object in DBStorage or in filestorage.